recipes/databases/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "databases" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0bd10cff1f073ae5d5354eb95d4d5d7f436516ce48dc5d10a7d80a785f77efee
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
```

Comment:
Try replacing this version selector with `skip: True  # [py<36]` in the `build` section.

(And the same for the `run` requirements.)

Reply:
oh thanks @chrisburr, now I remember that the python version selectors in the requirements are ok only for noarch stuff, is this correct?

Reply:
Yes you're correct. If `aiocontextvars` wasn't a Python 3.6 only dependency this could have been a `noarch` package and used `python >=3.6` instead.

Reply:
thanks again, sorry for bothering you